addons/sale/static/src/js/tours/sale.js

```diff
@@ -107,7 +96,7 @@ registry.category("web_tour.tours").add("sale_tour", {
             markup(_t("<b>Send the quote</b> to yourself and check what the customer will receive.")),
         ),
         {
-            isActive: ["body:not(:has(.modal-footer button[name='action_send_mail']))"],
+            isActive: ["body:has(.modal-footer button[name='document_layout_save'])"],
```

Comment:
I suppose you made sure it works fine by following it with two different users in the same database? :eyes: (the first one will have the configuration wizard if he's the admin but not the second one)

Reply:
Yep, I tested both flows :) But for some reason it stopped working since I first created my PR :sweat_smile:

Reply:
Should be fine now :)